app/views/admin/poll/polls/index.html.erb

```diff
@@ -14,6 +14,8 @@
       <th class="small-5"><%= t("admin.polls.index.name") %></th>
       <th class="text-center"><%= t("admin.polls.index.start_date") %></th>
       <th class="text-center"><%= t("admin.polls.index.closing_date") %></th>
+      <th class="text-center"><%= SDG::Goal.model_name.human(count: 2).upcase_first %></th>
+      <th class="text-center"><%= SDG::Target.model_name.human(count: 2).upcase_first %></th>
```

Comment:
This is (I think) a mistake I've made as well :sweat_smile:. Instead of `count: 2`, I think `count: :other` is better since there are languages which have a different plural rules depending on the number of elements :thinking:.